config/app/api/openapi/registry.yml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.0.0
+info:
+    version: 1.0.0
+    contact:
+        name: Spryker
+        url: 'https://apps.spryker.com/'
+        email: support@spryker.com
+    title: 'Spryker API'
+    license:
+        name: MIT
+paths:
+    '/private/application/config':
```

Comment:
Since we use this file for standalone App repos we can omit `application` this is only needed for our App Store Suite repo as we have multiple Apps in there.

Reply:
Shouldn't we use `configuration` instead of `config`?

Reply:
Don't we need 

```
security:
                -
                    BearerAuth: []
```

as well?

Reply:
Regarding security schema, this is actually a great question.
From my understanding, the authentication is handled outside the application (on the infrastructure level) an this raises the question of whether or not the OpenAPI spec should include infrastructure-level specifications, as it will only apply to spryker-hosted applications.

I think this should (at least) be optional (or opt-in) if we choose to include it.

Reply:
We usually send a bearer token in our requests and when that is the case we must reflect that in our schemas.

Reply:
Removed `application` suffix; added security details